sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/inc/argagg.hpp

```diff
@@ -0,0 +1,1541 @@
+/*
+ * @file
+ * @brief
+ * Defines a very simple command line argument parser.
+ *
+ * @copyright
+ * Copyright (c) 2018 Viet The Nguyen
```

Comment:
Anything similar that comes from within Microsoft?

Reply:
I'm not aware of any Microsoft authored command line parsers which do similar functionality.

This is the command line parser that Victor used for the perf tests. I stole it because I needed similar functionality and stole it (and then hacked it up a bit to allow -? as a switch).